transport_nantes/photo/forms.py

```diff
@@ -30,3 +31,59 @@ class Meta:
             'photographer_comments': forms.Textarea,
             'submitted_photo': forms.FileInput,
         }
+
+
+class AnonymousVoteForm(forms.Form):
+    """
+    Form to collect anonymous votes the first time they vote.
+    """
+    CHOICES = [("upvote", 'Pour'), ("downvote", 'Contre')]
+    vote_nature = forms.ChoiceField(
```

Comment:
This means something very different in English than in French.  It suggests a choice for greenery, for the natural order.  I'm afraid we need to rename this in order to avoid a persistent miscommunication on what is intended.  Just calling it "vote" would work.

Reply:
I don't want to name it vote because it would name conflict with the Vote object, hence the "nature" I appended to make a difference. 
It's important that the name doesn't make you question whether we're talking about the instance or its boolean

Reply:
vote_value ?

Reply:
Deal